syntax = "proto3";
package dfi;

message DFIFreeSegmentsRequest {
    string name = 1;
}
